src/main/java/com/ibm/disni/rdma/verbs/impl/NatPostSendCall.java

```diff
@@ -78,18 +86,10 @@ public void set(IbvQP qp, List<IbvSendWR> wrList) {
 			natSendWR.setPtr_sge_list(sgeOffset);
 			natSendWR.setNext(wrOffset);
 			wrNatList.add(natSendWR);
-			
-			size += NatIbvSendWR.CSIZE;
-			size += sendWR.getSg_list().size()*NatIbvSge.CSIZE;
+
```

Comment:
I don't get it. Below I can you you removed the free method in RdmaVerbsNat. But here in NatPostSendCall you're still calling that free method. Do you even check if the entire thing is compiling?!!

Reply:
Sorry forgot to push updates. Fixed all the stuff.

Reply:
I noticed this line as well in the previous PR. 
table = new ConcurrentSkipListMap<>();

I consider this to be bad practice, please can you add the type information there? If you can just make that modification in this PR.

Thanks!

Reply:
Done